MDANSE/Src/MDANSE/Framework/Configurators/AtomTransmutationConfigurator.py

```diff
@@ -147,11 +147,15 @@ def configure(self, value: str):
             self.error_status = "Key of transmutation map should be castable to int"
             return
 
-        for idx, element in value.items():
-            if idx not in idxs:
-                self.error_status = "Inputted setting not valid - atom index not found in the current system."
-                return
+        keys = {index for index in value.keys()}
```

Comment:
```suggestion
        keys = set(value.keys())
```
Although it should be noted that [`KeysView`s](https://docs.python.org/3/library/stdtypes.html#dict-views) behave like sets in most circumstances